### PR TITLE
created scripts for question and answer tables

### PR DESCRIPTION
**Pull request by m4ndycheung**

### CODE REVIEW

File: db-structure/create_answer.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE answer (
+  answer_id          SERIAL PRIMARY KEY,
+  option_position    INT NOT NULL,
```
Comment (malkitbenning) [inline db-structure/create_answer.sql:3]: Hi Mandy, just to match the other scripts, could you please change from INT to INTEGER? thanks

File: db-structure/create_answer.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  answer_text        VARCHAR(255) NOT NULL,
+  answer_score       INT NOT NULL
```
Comment (malkitbenning) [inline db-structure/create_answer.sql:5]: same here please - INTEGER instead of INT 
thanks

**Review by malkitbenning — CHANGES_REQUESTED** [review]
Hi Mandy, my bad! I should have used the word INTEGER on the diagram and not INT - doh!

File: db-structure/create_answer.sql (hunk 1 above)
Comment (m4ndycheung) [reply db-structure/create_answer.sql]: I've updated my file now, thanks for spotting!

**Comment by malkitbenning** [comment]
Script reviewed for syntax and correct field names. All good.

**Pull request merged**